lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -61,6 +62,11 @@ def run(path, contents)
           super([path])
         rescue RuboCop::Runner::InfiniteCorrectionLoop => error
           raise Formatting::Error, error.message
```

Comment:
`Formatting::Error` may not be appropriate here, since the error could occur either during linting _or_ formatting...

Reply:
Are you talking about `InfiniteCorrectionLoop`? That error can only occur when running with `--auto-correct` which we only do in formatting.

Reply:
Sorry, commented on the wrong line, I was referring to the new `raise` I added below.

Reply:
Oh, I see. We can create a parent class for the formatting and diagnostics error and then rescue from that parent class.